tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Placeholders.txt

```diff
@@ -203,6 +203,8 @@
 
 ^//ldml/numbers/currencyFormats/unitPattern ; {0}=NUMBER 3; {1}=CURRENCY_NAME Dollars
 ^//ldml/numbers/currencyFormats\[@numberSystem="%A"]/unitPattern ; {0}=NUMBER 3; {1}=CURRENCY_NAME Dollars
+^//ldml/numbers/currencyFormats/currencyPatternAppendISO ; {0}=CURRENCY_FORMAT ¤#,##0.00
```

Comment:
I don't see any seed data for `currencyPatternAppendISO`; is that intentional?

Reply:
There is a pattern in root. Locales can override it if they choose to.